src/main/httpCommandServer.ts

```diff
@@ -79,6 +107,43 @@ export default class HttpCommandServer {
 
     return true;
   }
+
+  protected lookupCommand(method: string, commandName: string) {
+    const commandsForMethod = this.dispatchTable[method];
+    const command = commandsForMethod ? commandsForMethod[commandName] : undefined;
+
+    return command && command.bind(this);
+  }
+
+  listSettings(request: http.IncomingMessage, response: http.ServerResponse) {
```

Comment:
Would it make sense to have `CommandWorkerInterface` have this directly? That is,
```ts
export interface CommandWorkerInterface {
  getSettings: (request: http.IncomingMessage, response: http.ServerResponse) => void;
  …
}
```
It feels a bit weird that the implementation of the handler is split into two places, but on the other hand maybe it's to delineate the HTTP-relevant part and the main process-relevant part?

Reply:
The idea is that a `CommandWorker` either returns a stringable value (and I guess a true/false status for a modifying action), or throws an error. And all that should happen in the HTTP server is unpacking requests, dispatching commands, and writing results to the response object.

`shutdown` is a bit of a special case because the server has to finish writing everything back to the response object before it carries out its associated business logic, as that logic is going to shut everything down, including the server.

Reply:
Okay, it would probably be a good idea to add a doc comment to `CommandWorkerInterface` to describe these design decisions so it's easier to understand (without looking at random GitHub comments).

Unresolving for that; there's no actual change at this line.

Reply:
Added doc comments, for the class defn as well as the interface defn.